src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +328,70 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const renderCustomBadge = () => {
```

Comment:
Still hoping https://github.com/elastic/kibana/pull/189404#discussion_r1700328195 can be addressed :) LMK if you have questions about it!

Reply:
I did! Thank you for helping me with it over zoom!